paasta_tools/chronos_serviceinit.py

```diff
@@ -167,6 +190,7 @@ def format_chronos_job_status(job, running_tasks, verbose):
     disabled_state = _format_disabled_status(job)
     (last_result, formatted_time) = _format_last_result(job)
     schedule = _format_schedule(job)
+    parents = _format_parents(job)
```

Comment:
Although probably not the simplest to do with how the code is now. Can you make this only show up in verbose mode?

As we use chronos more, paasta status is going to start getting really long. Most of the time people won't be interested in this data, and if they *are* inspecting a particular instance, *then* I think this detailed parent status will be useful.